tools/scylla-sstable.cc

```diff
@@ -2816,6 +2833,11 @@ for more information on this operation, including the API documentation.
                 typed_option<program_options::string_map>("script-arg", {}, "parameter(s) for the script"),
             }},
             script_operation},
+/* shard-of */
+    {{"shard-of",
+            "Print out the shard which 'owns' the sstable",
+            "Print out the intersection(s) of the shard-ranges and the partition ranges"},
+            shard_of_operation},
```

Comment:
I think we should have a `--text` (or --human-readeable or similar) option to output a human-friendly text output. The JSON output is hard to read without `jq` and the latter is not installed on every server.
I think we should go back and do the same with `validate` and `validate-checksums`, but that is for another PR.

Reply:
Maybe `--text|-t` is best, as it is short to type.

Reply:
> The JSON output is hard to read without `jq` and the latter is not installed on every server.

this applies to all subcommand which prints JSON. probably we should instead redistribute `jq` in the scylla image?

Reply:
i think this deserves a github issue.

Reply:
maybe yaml output is more readable (all json can be printed as yaml)

Reply:
tracked at #16361

Reply:
i will add the `-t|--text` option in a follow-up change.